js/VCRenderer.js

```diff
@@ -0,0 +1,29 @@
+const {DateTimeUtils} = require("./DateTimeUtils");
+class VCRenderer {
+    static renderSVG = async(data) => {
+
+        const templateUrl = data.renderMethod[0].id;
+        const response = await fetch(templateUrl, {
+            method: "GET",
+            headers: {
+                "Content-Type": "image/svg+xml"
+            }
+        });
+        const templateString = await response.text();
+        return templateString.replace(/{{(.*?)}}/g, (match, key) => {
+            key = key.replace(/^\//, '').replace(/\/$/, '');
+            const keys = key.split('/');
+            let value = data;
+            keys.forEach(k => {
+                if (value) {
+                    value = value[k];
+                    if (DateTimeUtils.isValidDateTime(value)) {
+                        value = DateTimeUtils.formatDate(value);
```

Comment:
We have to check this formatting should not get applied on numbers like policy number

Reply:
Since it checks for Date object, it should return false for policy number. Added in test case as well.